0800-appendix-198217-trib-to-skeena.Rmd

```diff
@@ -364,3 +395,61 @@ knitr::include_graphics(my_photo1)
 knitr::include_graphics("fig/pixel.png")
 knitr::include_graphics(my_photo2)
 ```
+
+<br>
+
+```{r photo-198217-07-prep}
+my_photo1 = fpr::fpr_photo_pull_by_str(site = my_site2, dir_photos = "data/photos/", str_to_pull = '_ct_us')
+
+my_caption1 = paste0('Cutthroat trout captured upstream of PSCIS crossing ', my_site2, '.')
+
+```
+
+```{r photo-198217-07, fig.cap= my_caption1, out.width = photo_width, eval=gitbook_on}
+grid::grid.raster(jpeg::readJPEG(my_photo1))
+```
+
+<br>
+
+```{r photo-198217-08-prep}
+my_photo2 = fpr::fpr_photo_pull_by_str(site = my_site2, dir_photos = "data/photos/", str_to_pull = '_dv_ds')
+
+my_caption2 = paste0('Dolly Varden captured downstream of PSCIS crossing ', my_site2, '.')
+
+```
+
+```{r photo-198217-08, fig.cap= my_caption2, out.width = photo_width, eval=gitbook_on}
+grid::grid.raster(jpeg::readJPEG(my_photo2))
+```
+
+<br>
+
+```{r photo-198217-09-prep}
+my_photo3 = fpr::fpr_photo_pull_by_str(site = my_site2, dir_photos = "data/photos/", str_to_pull = '_ct_ds')
+
+my_caption3 = paste0('Cutthroat trout captured downstream of PSCIS crossing ', my_site2, '.')
+
+```
+
+```{r photo-198217-09, fig.cap= my_caption3, out.width = photo_width, eval=gitbook_on}
+grid::grid.raster(jpeg::readJPEG(my_photo3))
+```
+
+<br>
+
+```{r photo-198217-u01, fig.cap = my_caption, fig.show="hold", out.width= c("49.5%","1%","49.5%"), eval=identical(gitbook_on, FALSE)}
+
+my_caption <- my_caption1
+knitr::include_graphics(my_photo1)
+```
+
+<br>
+
+```{r photo-198217-u02, fig.cap = my_caption, fig.show="hold", out.width= c("49.5%","1%","49.5%"), eval=identical(gitbook_on, FALSE)}
+
+my_caption <- paste0('Left: ', my_caption2, ' Right: ', my_caption3)
+
+knitr::include_graphics(my_photo2)
```

Comment:
convention with double photos for chunk name is `photo-1234-d04` because this is the 4th double photo in the pdf.  We should stick with that.  Also - I know it is annoying but lets keep the entire chunk as is and only include even numbers of photos so that we never need to doctor this code at all (ex. by including `my_caption3` and `my_photo3`).  Every photo on its own takes up a whole page of a pdf report and they are already hundreds of pages long.  Its rare that we can't make that work and often we can minimize the number of photos leaning towards less than more and still tell the whole story.

Reply:
okay that makes sense. Still getting the hang of all the photo stuff.